.travis/stages/verify/check-links-and-yaml-syntax/check-links

```diff
@@ -15,28 +17,31 @@ fi
 
 function checkUri() {
   local -r uri="$1"
-  
-  local -r curlResult=$(curl -s --head "$uri" | head -n 1)
-  echo "$uri -- $curlResult"  
-  # Temporarily disabled. We're unable to connect to A&A.org from Travis.
-
-  # echo $curlResult |  grep -qE  "HTTP/2 200|(200|302) (OK|Found)" \
-  #  || reportBrokenUri "$uri"
+  local output
+  if ! output="$FAILURE_PREFIX$(curl -Lsf -o /dev/null -w "%{http_code}" "$uri")"; then
```

Comment:
I think this is a lot more robust than the code we had before.
The command prints the actual HTTP status code purely for logging purposes.
`-L` makes sure to follow redirects.
`-f` makes curl return with non-zero exit codes in case the status code is 4xx or 5xx or something like that, whatever it considers to be an error
`-o /dev/null` just redirects the HTTP body i.e. the HTML to void so it won't land in stdout
`-w "%{http_code}"` makes curl print the status code in stdout if it exists, or "000" otherwise

Reply:
`-f` and `-w` are really good flags, good work :+1: 

I think a good chunk of this comment probably needs to land in the code so that this reasoning/logic explanation can be carried forward.

Reply:
Minor (consider fix if feasible): I also wonder if breaking up the assignment and status code check could potentially make things simpler, it seems we are quite overloaded on one line. Would it look gnarly to break it up to multiple lines?

Reply:
The primary reason I made it like this is because shellheck suggested it.

I see your point but I'm not actually sure it would make things better tbh